src/test/java/uk/nhs/tis/sync/job/PostFundingStatusSyncJobTest.java

```diff
@@ -0,0 +1,193 @@
+    assertThat(expectedQuery, is(actualQuery));
+  }
+
+  @Test
+  void testShouldConvertDataWithNoPostsAndFundings() {
+    List<EntityData> entityData = new ArrayList<>();
+    Set<Post> entitiesToSave = new HashSet<>();
+
+    int result = postFundingStatusSyncJob.convertData(entitiesToSave, entityData, entityManager);
+
+    assertThat(result, is(0));
+    assertThat(entitiesToSave.size(), is(0));
+  }
+
+  @Test
+  void testShouldPassWhenAPostHasMultiplePostFundings() {
+    Post post = new Post();
+    post.setId(1L);
+    PostFunding postFunding1 = new PostFunding();
+    postFunding1.setId(999L);
+    PostFunding postFunding2 = new PostFunding();
+    postFunding2.setId(1000L);
+    post.fundingStatus(Status.CURRENT);
+    Set<PostFunding> postFundingSet = new HashSet<>(Arrays.asList(postFunding1, postFunding2));
+    post.setFundings(postFundingSet);
+
+    when(entityManager.find(Post.class, post.getId())).thenReturn(post);
+
+    List<EntityData> entityData = new ArrayList<>();
+    EntityData entity = new EntityData();
+    entity.entityId(1L);
+    entityData.add(entity);
+    Set<Post> entitiesToSave = new HashSet<>();
+
+    int result = postFundingStatusSyncJob.convertData(entitiesToSave, entityData, entityManager);
+
+    assertThat(entitiesToSave.size(), is(1));
+    assertThat(result, is(0));
+    assertThat(entitiesToSave.contains(post), is(true));
+    assertThat(post.getFundingStatus(), is(Status.CURRENT));
+  }
+
+  @Test
+  void testJobShouldChangeCurrentPostFundingToInactive() {
+    Post post = new Post();
+    post.setId(1L);
+    PostFunding postFunding = new PostFunding();
+    postFunding.setId(999L);
+    post.fundingStatus(Status.CURRENT);
+    Set<PostFunding> postFundingSet = Collections.singleton(postFunding);
+    post.setFundings(postFundingSet);
+
+    when(entityManager.find(Post.class, post.getId())).thenReturn(post);
+
+    List<EntityData> entityData = new ArrayList<>();
+    EntityData entity = new EntityData();
+    entity.entityId(1L);
+    entityData.add(entity);
+
+    Set<Post> entitiesToSave = new HashSet<>();
+    int result = postFundingStatusSyncJob.convertData(entitiesToSave, entityData, entityManager);
+
+    assertThat(result, is(0));
+    assertThat(entitiesToSave.contains(post), is(true));
+    assertThat(post.getFundingStatus(), is(Status.INACTIVE));
+  }
+
+  @Test
+  void testShouldBeSuccessfulWithHandleDataMethod() {
+    Post post = new Post();
+    Set<Post> dataToSave = new HashSet<>();
+    dataToSave.add(post);
+    postFundingStatusSyncJob.handleData(dataToSave, entityManager);
+
+    verify(entityManager, times(1)).persist(post);
+    verify(entityManager, times(1)).flush();
+  }
+
+  @Test
+  void testElapsedTimeAndIsCurrentlyRunning() {
+    boolean running = postFundingStatusSyncJob.isCurrentlyRunning();
+    String runningTime = postFundingStatusSyncJob.elapsedTime();
+    assertFalse(running);
+    assertEquals("0s", runningTime);
```

Comment:
You've used hamcrest `assertThat()` in a number of places that `assertEquals()` could be used or vice versa.  You should aim to be consistent with what you use.

Reply:
switched to assertThat()